sdk/firstload/src/integrations/amplitude.ts

```diff
@@ -1,6 +1,6 @@
 // @ts-nocheck
```

Comment:
not sure why we have `@ts-nocheck` here but does your change mean we can remove the override?

Reply:
Doesn't look like it. When I removed the `@ts-nocheck`, 49 errors end up showing up when running tsc.

It looks like these were mostly here to silence type errors from what looks like copy pasted init scripts?

Reply:
ah gotcha, this is for the file rather than the import. thanks!